src/csharp/Grpc.Core.Api/ChannelCredentials.cs

```diff
@@ -51,6 +52,23 @@ public static ChannelCredentials Insecure
             }
         }
 
+        /// <summary>
+        /// Returns instance of credentials that provides SSL security.
+        /// <para>
+        /// When used with Grpc.Core, these credentials will load from a
```

Comment:
Let's rephrase this paragraph to say that using ChannelCredentials.Secure has the same effect as using a default instance of parameterless SslCredentials() (the "default" SSL credentials).
Also add a note that for Grpc.Core, one should avoid using this shortcut if want to specify extra configuration, such as custom trust roots, mutual TLS or specify CallCredentials.
Copypasting the documentation of SslCredentials() parameterless constructor here is not necessary.

Reply:
Done